### PR TITLE
13058 COLIN API failure - Front and Backend Change validation of the first name field in Coop IA UI and/or Corp Party table

### PR DESCRIPTION
**Pull request by lewischenstudio**
*Issue #:* [/bcgov/entity#13058](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/13058)

*Description of changes:*
- change the max length of "firstName" to 20
- change UI before the schema change

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Review by argush3 — APPROVED** [review]

**Review by severinbeauvais — APPROVED** [review]
LGTM.

Let me know when you need help publishing this change.

Also, you'll have to import this new component in the UI(s).

**Comment by lewischenstudio** [comment]
> LGTM.
> 
> Let me know when you need help publishing this change.
> 
> Also, you'll have to import this new component in the UI(s).

Sure. Thanks. I'll update the UIs.

File: src/components/completing-party/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bcrs-shared-components/completing-party",
-  "version": "2.0.3",
+  "version": "2.0.4",
```
Comment (severinbeauvais) [inline src/components/completing-party/package.json:3]: This gets done automatically when by `lerna version --include-merged-tags`. Refer back to the readme for this project. Please revert this change.

Comment (severinbeauvais) [reply src/components/completing-party/package.json:3]: OK, since you've merged this already, I will fix it when I run steps 14-15 in the readme.

Comment (argush3) [reply src/components/completing-party/package.json:3]: actually, @lewischenstudio can revert it in his upcoming PR to update the middle name length

Comment (severinbeauvais) [reply src/components/completing-party/package.json:3]: I can't update it without pushing another commit, so I just went with the next version number.

Version 2.0.5 is now ready for import into other UIs.
